hello

```diff
@@ -0,0 +1 @@
+nihao
```

Comment:
Maybe we should use a more internationalized way to do this

Reply:
Resolved.